Back-End-Node/server.js

```diff
@@ -1,23 +1,126 @@
 const express = require("express");
 const app = express();
 const { pool } = require("./dbConfig");
-
+var http = require("http");
```

Comment:
we need to change all "var"s with const or let, this really old version of declaring a variable and is not used in JavaScript any more, but i think you took it from C# , did not you :wink:

Reply:
It is just copy paste from some another examples. I will change those with next pull request.